src/js/pouchdb-node.js

```diff
@@ -94,10 +98,53 @@ gpii.pouch.node.loadDataIfNeeded = function (that) {
     }
 };
 
+/**
+ *
+ * Remove all content, close, and then remove the directory content for this database.
+ *
+ * @param that
+ * @param callback
+ * @returns {*}
+ */
+gpii.pouch.node.cleanPouch = function (that, callback) {
+    var promise = fluid.promise();
+    gpii.pouch.cleanPouch(that, function (cleanupMessage) {
+        if (!cleanupMessage.ok) {
+            promise.reject(cleanupMessage);
+        }
+        else {
+            that.close().then(function (closeError) {
+                if (closeError) {
+                    promise.reject(closeError);
+                }
+                else {
+                    if (that.options.removeDirOnCleanup) {
+                        rimraf(that.options.dbOptions.prefix, function (rmDirError) {
```

Comment:
This recursively removes everything in the directory of `that.options.dbOptions.prefix`, which in this case is everything in the system temp directory, instead of only the database specific directory.